### PR TITLE
Eliminated all linter issues

### PR DESCRIPTION
**Pull request by mphprogrammer**
@saneshark With regards to message_params, my JHS french teach, Mr Grebanier, had a saying:
```
If you're a cop writing a report about a dead horse, and the horse died on Kosciuszko Street
 - drag it to Dean Street.
```
I dragged the horse to Dean Street.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -26,13 +26,26 @@
...
+      scope :health do
+        resources :triage_teams, only: [:index], defaults: { format: :json }, path: 'recipients'
+
+        resources :folders, only: [:index, :show], defaults: { format: :json } do
+          resources :messages, only: [:index], defaults: { format: :json }
+        end
+
+        resources :messages, only: [:show, :create, :destroy], defaults: { format: :json } do
+          post :draft, on: :collection
+          put :draft, on: :member
```
Comment (saneshark) [inline config/routes.rb]: what are the differences between these two?

File: app/controllers/v0/messages_controller.rb [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+      render json: response.data[0],
+             serializer: MessageSerializer,
+             meta: response.data[0].metadata
+    end
+
+    def create
+      params = message_params
+      Rails.logger.error "@@@@@@@@@@@ #{params.inspect}"
+      response = client.post_create_message(subject: params[:subject], body: params[:body], id: params[:id],
```
Comment (saneshark) [inline app/controllers/v0/messages_controller.rb]: create should never accept an id

File: app/controllers/v0/messages_controller.rb [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      raise VA::API::Common::Exceptions::RecordNotFound, message_id unless response.present?
+
+      render json: response.data[0],
+             serializer: MessageSerializer,
+             meta: response.data[0].metadata
+    end
+
+    def create
+      params = message_params
+      Rails.logger.error "@@@@@@@@@@@ #{params.inspect}"
```
Comment (saneshark) [inline app/controllers/v0/messages_controller.rb]: probably don't want this

Comment (mphprogrammer) [inline app/controllers/v0/messages_controller.rb]: LOL No I don't!

File: config/routes.rb [hunk 4]
```diff
@@ -26,13 +26,26 @@
...
+    scope :messaging do
+      scope :health do
+        resources :triage_teams, only: [:index], defaults: { format: :json }, path: 'recipients'
+
+        resources :folders, only: [:index, :show], defaults: { format: :json } do
+          resources :messages, only: [:index], defaults: { format: :json }
+        end
+
+        resources :messages, only: [:show, :create, :destroy], defaults: { format: :json } do
+          post :draft, on: :collection
```
Comment (saneshark) [inline config/routes.rb]: can we also remove any unused routes?

File: app/controllers/v0/messages_controller.rb (hunk 2 above)
Comment (mphprogrammer) [inline app/controllers/v0/messages_controller.rb]: Create with an id sends a draft.

Comment (saneshark) [inline app/controllers/v0/messages_controller.rb]: is there some way of abstracting that out -- doesn't seem like something intuitive for an api user.

**Comment by saneshark** [comment]
LGTM -- we will look to address some complexity around working with drafts and deleting messages in a separate PR.

**Pull request merged**